server/cmd/mmctl/commands/auth_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+func (s *MmctlUnitTestSuite) TestAuthList() {
+	originalUser := *currentUser
+	defer func() {
+		SetUser(&originalUser)
+	}()
+
+	tmp, _ := os.MkdirTemp("", "mmctl-")
+	defer os.RemoveAll(tmp)
```

Comment:
Nit pick: In general, it's advised to use `s.T().Cleanup` to cleanup after the test run.